### PR TITLE
Updating examples for scrollbar gutter

### PR DESCRIPTION
**Pull request by Rumyra**
#### Summary

Part of and as per @hamishwillee 's comment here https://github.com/mdn/content/issues/11595 adds images to visually show the differences in `scrollbar-gutter` values

#### Metadata

- [ ] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/css/scrollbar-gutter/index.md [hunk 1]
```diff
@@ -55,41 +56,58 @@ scrollbar-gutter: unset;
 
 ## Examples
 
+The examples below show how the different values for the `scrollbar-gutter` property would affect a scrollable `div` element (`.container`) with one of more paragraphs contained within.
+
+> **Note:** In the images for the examples, the users system settings are set to classic scrollbars (always shown).
+
```
Comment (sideshowbarker) [inline files/en-us/web/css/scrollbar-gutter/index.md:62]:
```suggestion
The examples below show how the different values for the `scrollbar-gutter` property would affect a scrollable `div` element (`.container`) with one or more paragraphs contained within.

> **Note:** In the images for the examples, the user’s system settings are set to classic scrollbars (always shown).

```

File: files/en-us/web/css/scrollbar-gutter/index.md [hunk 2]
```diff
@@ -55,41 +56,58 @@ scrollbar-gutter: unset;
...
+.container2 {
+  scrollbar-gutter: stable;
 }
 ```
 
+![Two adjacent divs containing text, both with space for a scrollbar](side-by-side.png)
+
+### Overlay scrollbars
+
+For reference this image shows the same div as above, but with the user system settings set to overlay scrollbars. Note here the scrollbar will only show when the user is scrolling and on top of the content, so no space is reserved for it and the `scrollbar-gutter` property has no effect.
```
Comment (sideshowbarker) [inline files/en-us/web/css/scrollbar-gutter/index.md:107]:
```suggestion
For reference, this image shows the same div as above, but with the user’s system settings set to overlay scrollbars. Note here the scrollbar will only show when the user is scrolling and on top of the content, so no space is reserved for it and the `scrollbar-gutter` property has no effect.
```

**Review by sideshowbarker — APPROVED** [review]

**Comment by hamishwillee** [comment]
Thanks @Rumyra. It turns out I did not understand how the scrollbar gutter worked, so the images did help!

**Comment by Rumyra** [comment]
Ha yes @hamishwillee it was a good call 👍 I think the examples are improved - and also I was just happy it worked in FF so i could make images 😂

**Pull request merged**